### PR TITLE
[Website] Improved install snippet. Newbie friendly samples. Code snippet styling, button styling, figure styling.

### PR DESCRIPTION
**Pull request by gnat**
A number of smaller refinements.

* Install snippet now wraps nicely on line overflow. Little changing icon that congratulates you :tada: 
* Made front-page samples even more newbie friendly / copy-pasteable.
* Consistent code snippet styling. Figure styling.

## BEFORE / AFTER Screenshot
image

### CODE REVIEW

**Comment by dz4k** [comment]
- The "try me" button being so prominent messes up the information hierarchy and makes the top section hard to parse. I end up reading the button before anything else and being confused.
- The code snippets being "consistent" is a downgrade imo. It's an interactive example, unlike the others, and it's at the top jumbotron area. It _should_ be set off from the rest. And again, the code snippet at the top being so muted messes up the information hierarchy/reading order, my eyes glazed over it
- Try-me button encroaches too far into code block since padding has been reduced
- The copy button sometimes wraps to the next line. Again, too prominent
- Adding a background to the code examples is something I'm ambivalent about, but the background shouldn't extend past the end of the code as it does in "Remember HyperCard"
- You've modified the appearance of `pre` elements, buttons, as well as `html figure, .box, aside, details, dialog` (i'm assuming that's why the admonition at the bottom looks different, can't trace the code) (not sure why there's `html` in front of `figure`). These changes should be PRed against missing.css mainline rather than monkey-patched here.
- Relatedly, any customizations should try to follow missing.css conventions (i.e. `button.special` should be `strong button`).

**Comment by gnat** [comment]
The front page, particularly before yesterday, was challenging for users, especially for newcomers. I'm attempting to add some of the good aspects of the classic design into your new one.

> The "try me" button being so prominent messes up the information hierarchy and makes the top section hard to parse. I end up reading the button before anything else and being confused.

The old one did not look like a button you could press- not even a "clickable" pointer on hover.

> The code snippets being "consistent" is a downgrade imo.  

The design can afford consistency without a disadvantage, so why not have consistency. Subjective opinions here of course, I mean we could make everything a unique color rainbow if we wanted.

> Try-me button encroaches too far into code block since padding has been reduced.

I've increased the distance. I am indifferent and don't think it makes a huge difference though.

> The copy button sometimes wraps to the next line. Again, too prominent

The wrap acts fine in this current PR version. Any line this long is going to have a break.

> but the background shouldn't extend past the end of the code as it does in "Remember HyperCard"

Fixed, thank you.

> You've modified the appearance of pre elements, buttons, as well

Yeah, just got rid of some of the unnecessary borders in the design. The warning at the bottom was taking a ton of attention, and worked alright to mellow it out.

> Relatedly, any customizations should try to follow missing.css conventions (i.e. button.special should be strong button).

Does strong even make sense here? The only difference is size, used only one time in the header, hence `button.special`. There's merit in keeping it simple.

## Revision 2
image
